unittests/test_policies.py

```diff
@@ -466,7 +482,7 @@ def assert_dependency_run(runner):
     assert_runall(runner)
     stats = runner.stats
     assert 10 == stats.num_cases(0)
-    assert 4  == len(stats.failed())
+    assert 4 == len(stats.failed())
```

Comment:
Revert the formatting :-) I think you should tell your editor to ignore E221 and W504.